types/mongodb/index.d.ts

```diff
@@ -1426,6 +1426,25 @@ export type UpdateQuery<TSchema> = {
     $bit?: {
         [key: string]: { [key in 'and' | 'or' | 'xor']?: number };
     };
+} | Array<UpdateAggregation<TSchema>>;
+
+/** https://docs.mongodb.com/v4.2/release-notes/4.2/#update-enhancements */
+export type UpdateAggregation<TSchema> = {
```

Comment:
This union has no obvious discriminants. How will users tell whether they have, for example, `$replaceRoot` or `$set`? How will Typescript know?